app/scripts/metamask-controller.js

```diff
@@ -2597,7 +2596,6 @@ export default class MetamaskController extends EventEmitter {
   }
 
   stopNetworkRequests() {
```

Comment:
We probably also want to call `this.accountTrackerController.stopAllPolling();` in `onClientClosed`, just to be sure no polling loops leak, similar to the other polling controllers.

Reply:
Makes sense! [9fc9319](https://github.com/MetaMask/metamask-extension/pull/28277/commits/9fc931961cb36fde7e8d6e375262c63fadf94767)